static/app/components/devtoolbar/components/featureFlags/featureFlagsPanel.tsx

```diff
@@ -1,82 +1,152 @@
+            </AnalyticsProvider>
+          </div>
+          <div css={[smallCss, panelSectionCss, panelInsetContentCss]}>
+            <AnalyticsProvider keyVal="custom-override" nameVal="Custom Override">
+              <CustomOverride />
+            </AnalyticsProvider>
+          </div>
+        </div>
+      </PanelLayout>
+    </FeatureFlagsContextProvider>
+  );
+}
+
+function IsDirtyMessage() {
+  const {isDirty} = useFeatureFlagsContext();
 
+  return <span>{isDirty ? 'Reload to see changes' : ''}</span>;
+}
+
+function Filters({
+  setPreFilter,
+  prefilter,
+  setSearchTerm,
+}: {
+  prefilter: Prefilter;
+  setPreFilter: Dispatch<SetStateAction<Prefilter>>;
+  setSearchTerm: Dispatch<SetStateAction<string>>;
+}) {
+  const {clearOverrides} = useFeatureFlagsContext();
   return (
-    <PanelLayout title="Feature Flags">
-      <div css={[smallCss, panelSectionCss, panelInsetContentCss]}>
-        <span>
-          Flags enabled for <code>{organizationSlug}</code>
-        </span>
+    <Fragment>
+      <div css={{gridArea: 'segments'}}>
+        <SegmentedControl<Prefilter> onChange={setPreFilter} size="xs" value={prefilter}>
+          <SegmentedControl.Item key="all">All Flags</SegmentedControl.Item>
+          <SegmentedControl.Item key="overrides">Overrides Only</SegmentedControl.Item>
+        </SegmentedControl>
       </div>
-      <PanelTable
-        headers={[
-          <div key="column">
-            <Input
-              ref={searchInput}
-              size="sm"
-              placeholder="Search flags"
-              onChange={e => setSearchTerm(e.target.value.toLowerCase())}
-            />
-          </div>,
-        ]}
-        stickyHeaders
-        css={[
-          {
-            border: 'none',
-            padding: 0,
-            '&>:first-child': {
-              minHeight: 'unset',
-              padding: 'var(--space50) var(--space150)',
-            },
-          },
-        ]}
+      <Button
+        size="xs"
+        onClick={() => {
+          clearOverrides();
+        }}
+        css={{gridArea: 'clear'}}
       >
-        {filteredItems?.length ? (
-          filteredItems.map(flag => {
-            return (
-              <Cell key={flag} css={[panelInsetContentCss, {alignItems: 'flex-start'}]}>
-                {featureFlagTemplateUrl?.(flag) ? (
-                  <AnalyticsProvider nameVal="item" keyVal="item">
-                    <ExternalLink
-                      css={[smallCss, inlineLinkCss]}
-                      href={featureFlagTemplateUrl(flag)}
-                    >
-                      {flag}
-                    </ExternalLink>
-                  </AnalyticsProvider>
-                ) : (
-                  <span>{flag}</span>
-                )}
-              </Cell>
-            );
-          })
-        ) : (
-          <EmptyStateWarning small>No items to show</EmptyStateWarning>
-        )}
-      </PanelTable>
-    </PanelLayout>
+        Clear Overrides
```

Comment:
could we have some sort of success indicator/toast at the bottom of the page when "clear overrides" is pressed?

Reply:
<img width="337" alt="SCR-20240731-kihi" src="https://github.com/user-attachments/assets/d522b3ef-f1c4-477c-ba3a-b53bc62158f4">
nit: there's also this extra section when the "reload to see changes" isn't there

Reply:
Toasts are hard right now because of the css/menu problem inside the shadowRoot. So i'm not going to fix that right away.

I'll move the condition so the empty space goes away though, but generally we're ok with some ui bugs like this in the toolbar!